e2e/testdata/fn-render/invalid-subpkg/Kptfile

```diff
@@ -0,0 +1,14 @@
+apiVersion: kpt.dev/v1alpha2
+kind: Kptfile
+metadata:
+  name: app
+pipeline:
+  mutators:
+  - image: gcr.io/kpt-fn/starlark:unstable
```

Comment:
General note, I think we should use versioned docker images instead of unstable. Otherwise, a bad function push can break all the test cases.

Reply:
Good point. Will make a sweeping change as part of m2 milestone, I think at that point, we will have basic functions versioned as well.